go/interpreter/lfvm/interpreter_test.go

```diff
@@ -375,7 +375,7 @@ func TestRunWithLogging(t *testing.T) {
 		runner: loggingRunner{},
 	}, params, code)
 	// read the output
-	w.Close()
+	_ = w.Close() // ignore error in test
```

Comment:
Why not do the usual in tests - check for an error and fail if there is one? 
There is no need for 100% code coverage in tests.

Reply:
This is a funny scenario:
the close happens between `_, err := run(interpreterConfig{` and `if err != nil` (for the run err) 

This is the precise function call that made the decision not to enable this check inside of Vet. 

Because `err:= run` `err2:= close`  is pretty odd, I decided to ignore it for now. Real use cases should add a `defer` and deal with releasing resources properly.